include/multipass/platform.h

```diff
@@ -26,13 +26,18 @@
 
 #include <libssh/sftp.h>
 
+#include <QString>
+
 #include <string>
 
 namespace multipass
 {
 namespace platform
 {
 std::string default_server_address();
+QString default_driver();
+QString daemon_config_home(); // temporary
+bool is_backend_supported(const QString& backend); // temporary
```

Comment:
What is your plan for these on the other platforms?

Reply:
For 38 and 40 I was planning to maintain the behavior I find there. So, for instance, on win accept "hyperv" and "virtualbox" (keeping with the lowercase theme, but we could of course accept capitals as well if prefered). 

As for 39, I was planning to do something analogous to what we did here. That is, find the folder where the settings would naturally be saved by the daemon if we were already using RPC and have the client save them there, requiring admin privileges if necessary. 

How does that sound?

Reply:
On a more practical note, now would probably be the right time to get the windows install back up to speed in my other machine. As for mac, I would need help testing or remote access.

Reply:
Ok ,works for me.  Just wanted to make sure there was a plan in place;)